panel/src/components/Forms/Previews/HtmlFieldPreview.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="k-html-field-preview" :class="$options.class">
+    {{ column.before }}
+    <!-- eslint-disable-next-line vue/no-v-html -->
+    <div v-html="html" />
```

Comment:
Can’t we just use `value` here and get rid of the computed prop?

Reply:
I used the same method here that's also been used in the text field preview. This way you can extend the component and simply overwrite the computed property without having to change the template.